aio/aio-proxy/aio_proxy/response/unite_legale_model.py

```diff
@@ -0,0 +1,130 @@
+    cedex: str | None = None
+    code_pays_etranger: str | None = None
+    code_postal: str | None = None
+    commune: str | None = None
+    complement_adresse: str | None = None
+    coordonnees: str | None = None
+    date_creation: str | None = None
+    date_debut_activite: str | None = None
+    departement: str | None = None
+    distribution_speciale: str | None = None
+    est_siege: bool = None
+    etat_administratif: str | None = None
+    geo_adresse: str | None = None
+    geo_id: str | None = None
+    indice_repetition: str | None = None
+    latitude: str | None = None
+    libelle_cedex: str | None = None
+    libelle_commune: str | None = None
+    libelle_commune_etranger: str | None = None
+    libelle_pays_etranger: str | None = None
+    libelle_voie: str | None = None
+    liste_enseignes: list | None = None
+    liste_finess: list | None = None
+    liste_id_bio: list | None = None
+    liste_idcc: list | None = None
+    liste_id_organisme_formation: list | None = None
+    liste_rge: list | None = None
+    liste_uai: list | None = None
+    longitude: str | None = None
+    nom_commercial: str | None = None
+    numero_voie: str | None = None
+    region: str | None = None
+    siret: str = None
+    tranche_effectif_salarie: str | None = None
+    type_voie: str | None = None
+
+
+class UniteLegaleElu(BaseModel):
+    nom: str | None = None
+    prenoms: str | None = None
+    annee_de_naissance: str | None = None
+    fonction: str | None = None
+    sexe: str | None = None
+
+
+class UniteLegaleDirigeantsPM(BaseModel):
+    siren: str = None
+    denomination: str | None = None
+    sigle: str | None = None
+    qualite: str | None = None
+    type_dirigeant: Literal["personne morale"]
+
+
+class UniteLegaleDirigeantsPP(BaseModel):
+    nom: str | None = None
+    prenoms: str | None = None
+    annee_de_naissance: str | None = None
+    qualite: str | None = None
+    type_dirigeant: Literal["personne physique"]
+
+
+class UniteLegaleFinances(BaseModel):
+    annee_cloture_exercice: str | None = None
+    ca: int | None = None
+    resultat_net: int | None = None
+
+
+class UniteLegaleCollectiviteTerritoriale(BaseModel):
+    code: str | None = None
+    code_insee: str | None = None
+    elus: list[UniteLegaleElu] | None = None
+    niveau: str | None = None
+
+
+class UniteLegaleComplements(BaseModel):
+    collectivite_territoriale: UniteLegaleCollectiviteTerritoriale | None = None
+    convention_collective_renseignee: bool = None
+    egapro_renseignee: bool = None
+    est_bio: bool = None
+    est_entrepreneur_individuel: bool = None
+    est_entrepreneur_spectacle: bool = None
+    est_ess: bool = None
+    est_finess: bool = None
+    est_organisme_formation: bool = None
+    est_qualiopi: bool = None
+    liste_id_organisme_formation: list | None = None
+    est_rge: bool = None
+    est_service_public: bool = None
+    est_societe_mission: bool = None
+    est_uai: bool = None
```

Comment:
I am a bit suprised by these default values : 

`est_uai: bool = None` odes it means that default is `None` ? Shouldnt it be `False' ?

Reply:
Yes, you're right !